.pre-commit-config.yaml

```diff
@@ -58,6 +58,12 @@ repos:
         # MD025: single title
         args: [--disable, MD013, MD024, MD025, MD033, MD041, "--"]
 
+  - repo: https://github.com/tcort/markdown-link-check
```

Comment:
Need more tweak on this, it seems current hooks is super slow:

Without this hook:
```
pre-commit run --all-files  19.54s user 1.65s system 198% cpu 10.665 total
```


With this hook (hook installed beforehand):
```
pre-commit run --all-files  20.65s user 1.80s system 72% cpu 30.948 total
```